process/interceptors/disabledWhiteListDataVerifier.go

```diff
@@ -0,0 +1,32 @@
+package interceptors
+
+import (
+	"github.com/ElrondNetwork/elrond-go/process"
+	"github.com/ElrondNetwork/elrond-go/storage"
+)
+
+type disabledWhiteListVerifier struct {
+}
+
+// NewDisabledWhiteListDataVerifier returns a default data verifier
+func NewDisabledWhiteListDataVerifier(_ storage.Cacher) (*disabledWhiteListVerifier, error) {
```

Comment:
We usually omit the not required parameters on disabled implementations.

Reply:
done